src/components/one-time-donation/components/Steps.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import FirstStep from './FirstStep'
+import SecondStep from './SecondStep'
+import ThirdStep from './ThirdStep'
+import Success from './Success'
+import Unsuccess from './Unsuccess'
+import {
+  DonationBankInput,
+  DonationResponse,
+  DonationStep as StepType,
+  OneTimeDonation,
+} from '../../../gql/donations'
+import { FormikStep, FormikStepper } from './FormikStepper'
+import { useTranslation } from 'next-i18next'
+import { useRouter } from 'next/router'
+import { useMutation, UseQueryResult } from 'react-query'
+import { CampaignResponse } from 'gql/campaigns'
+import { useViewCampaign } from 'common/hooks/campaigns'
+import { useVaultsList } from 'common/hooks/vaults'
+import { useCreateBankDonation } from 'service/donation'
+import { AxiosError, AxiosResponse } from 'axios'
+import { ApiErrors, isAxiosError, matchValidator } from 'service/apiErrors'
+import { FormikHelpers } from 'formik'
+import { validateFirst, validateSecond, validateThird } from '../helpers/validation-schema'
+
+const initialValues: OneTimeDonation = {
+  message: '',
+  anonymous: false,
+  amount: 0,
+  anonymousDonation: true,
+  personsFirstName: '',
+  personsLastName: '',
+  personsEmail: '',
+  personsPhone: '',
+  payment: 'bank',
+}
+
+export default function DonationStepper() {
+  const [status, setStatus] = React.useState(false)
```

Comment:
What does `status: true` mean in this context?

Maybe we can bring more context if we rename it to `donationSuccess` or just `success` as it implies that it has only true or false as values.

Status fields normally have specific values like `pending`, `valid` and so on